rosa/cli.py

```diff
@@ -47,16 +49,19 @@ def hash_log_keys(log):
 
 def rosa_login(env, token, aws_region, allowed_commands=None):
     _allowed_commands = allowed_commands or parse_help()
-    if is_logged_in(allowed_commands=_allowed_commands, aws_region=aws_region):
+
+    try:
+        is_logged_in(allowed_commands=_allowed_commands, aws_region=aws_region, env=env)
         LOGGER.info(f"Already logged in to {env} [region: {aws_region}].")
         return
 
-    build_execute_command(
-        command=f"login --env={env} --token={token}",
-        allowed_commands=_allowed_commands,
-    )
-    if not is_logged_in(allowed_commands=_allowed_commands, aws_region=aws_region):
-        raise NotLoggedInError("Failed to login to AWS.")
+    except NotLoggedInOrWrongEnvError as ex:
+        LOGGER.error(ex)
```

Comment:
No need for logger here since we expect to fail if not logged in, we will have error if line 64 fail